config/serving.yaml

```diff
@@ -40,6 +40,14 @@ repositories:
         - ".*e2e-tls$"
       onDemand:
         - "perf-tests$"
+      filterImages:
+        "perf-test":
+          - "knative-serving-load-test"
+          - "knative-serving-dataplane-probe"
+          - "knative-serving-real-traffic-test"
+          - "knative-serving-reconciliation-delay"
+          - "knative-serving-rollout-probe"
+          - "knative-serving-scale-from-zero"
```

Comment:
After integrating https://github.com/openshift-knative/hack/pull/84 it would be something like this:
```
  e2e:
      - match: ".*e2e$"
      - match: ".*e2e-tls$"
      - match: "perf-tests$"
        onDemand: true
        filterImages:
          - "knative-serving-load-test"
```

Reply:
I am fine rebasing, I will adapt mine, no worries.